test/univariate/continuous.jl

```diff
@@ -99,6 +99,19 @@ end
     @test skewness(d) ≈ 3β/(α*sqrt(δ*g))
 end
 
+@testset "InverseGaussian cdf outside of [0, 1] (#1873)" begin
+    for d in [
+        InverseGaussian(1.65, 590),
+        InverseGaussian(0.5, 1000)
+    ]
+        for x in [0.02, 1.0, 20.0, 300.0]
+            p = cdf(d, x)
+            @test 0.0 <= p <= 1.0
+            @test p ≈ exp(logcdf(d, x))
+        end
+    end
+end
+
```

Comment:
Can you please move the test to test/univariate/continuous/inversegaussian.jl?

Reply:
That was my original plan, but that file doesn't currently exist, and introducing it in this pull request would probably cause a merge conflict with this other recent pull request of mine, #1879 , as that one also introduces that file.

Reply:
I had missed #1879 but to get this PR in I'd suggest creating the test file anyway and then fixing the merge conflicts in #1879 (which should be easy to resolve).

Reply:
Sounds good.